README.md

```diff
@@ -846,7 +847,7 @@ Parameters:
 
 ```
 formatString [string] Format string based on moment.js (Required)
-date [date] The date/date-time that needs to be formatted. (Required)
+date [date] The date/date-time that needs to be formatted. (set to current Date() if not provided)
```

Comment:
```suggestion
date [date] The date/date-time that needs to be formatted. (defaults to current `Date()` if not provided)
```

Reply:
Ignore this, you should update the existing helper documentation after adding a new param.